dibi/drivers/DibiPdoDriver.php

```diff
@@ -78,7 +78,10 @@ public function connect(array & $config)
 		}
 
 		$this->driverName = $this->connection->getAttribute(PDO::ATTR_DRIVER_NAME);
-		$this->serverVersion = $this->connection->getAttribute(PDO::ATTR_SERVER_VERSION);
+		$this->serverVersion = @$this->connection->getAttribute(PDO::ATTR_SERVER_VERSION); // intentionally @
```

Comment:
~~~php
$this->serverVersion = isset($config['version'])
	? $config['version']
	: @$this->connection->getAttribute(PDO::ATTR_SERVER_VERSION);
~~~